test/wt_hang_analyzer/wt_hang_analyzer.py

```diff
@@ -447,6 +447,13 @@ def get_hang_analyzers():
 
     dbg = None
     ps = None
+
+    # Currently there are issues with MacOS and windows implementation. So, skip
+    # taking the dump and result in an error.
+    # FIXME : Remove the skip block of code after fixing the issues.
+    if _IS_WINDOWS or sys.platform == "cygwin" or sys.platform == "darwin":
+        return [ps, dbg]
```

Comment:
Skipping Windows this way will make the core file generation part unavailable, which is undesired. I'd rather not skipping Windows to let the core generated with a failure debugger.

Reply:
Fixed, now the script generates the dump in the Windows machine.